src/poliastro/bodies.py

```diff
@@ -52,9 +66,68 @@ def _repr_latex_(self):
 
         """
         return self.__str__()
+    def calculate_soi(self):
+        '''Calculate the radius of the sphere of influence.
+        The body must have a parent body and an orbit defined'''
+        if self.parent == None:
+            warnings.warn('Unable to calculate SOI without valid parent body')
+        elif self.orbit == None:
+            warnings.warn('Unable to calculate SOI without valid orbit assigned')
+        else:
+            self.soi= self.orbit.a * (self.k / self.parent.k) ** (2/5)
+
+_data_path = os.path.join(os.path.dirname(__file__), "bodies_data.txt")
+with open(_data_path, 'r', encoding='utf-8') as _raw_data :
+    for _line in _raw_data:
+        _line = _line.replace('\ufeff','')
+        _line = _line.replace('\n','')
+        if _line[0] == '#':
+            continue
+        _line = _line.replace(' ','')
+        _body_data = _line.split(sep = ',')
+        _name, _symbol, _k, _R = _body_data[:4]
+        _k = float(_k) * u.km**3 / u.s**2
+        _R = float(_R) * u.km
+        
+        if _body_data[4] == 'None':
+            _parent = None
+        else:
+            _parent_name = _body_data[4]
+            if _parent_name in body_dict:
+                _parent = body_dict[_parent_name]
+            else:
+                _message = 'loading bodies warning: '
+                _message += 'object not found during body loading:'
+                _message += _parent_name +'. Parent reverted to None for ' + _name
+                warnings.warn(_message)
+                _parent = None
+        if _body_data[5] == 'None':
+            _orbit = None
+        elif _parent == None:
+            _orbit = None
+            warnings.warn('Unable to create orbit without valid parent body')
+        else:
+            _a, _ecc, _inc, _L, _long_peri, _raan = _body_data[5:11]
+            _a = float(_a) * au
+            _ecc = float(_ecc) * u.one
+            _inc = (float(_inc) * u.deg).to(u.rad)
+            _L = (float(_L) * u.deg).to(u.rad)
+            _long_peri = (float(_long_peri) * u.deg).to(u.rad)
+            _raan = (float(_raan) * u.deg).to(u.rad)
+            _argp = _long_peri - _raan
+            _M = _L - _long_peri
+            _nu = M_to_nu(_M, _ecc)
+            _orbit = Orbit.from_classical(_parent, _a, _ecc, _inc, _raan, _argp, _nu)
+        body_dict[_name] = Body.from_parameters(_k, _name, _symbol, _R, _parent, _orbit)
+    
+
+for _body_name in body_dict:
+    _body = body_dict[_body_name]
+    if _body.parent != None and _body.orbit != None:
+        _body.calculate_soi()
+
+Sun = body_dict['Sun']
```

Comment:
I don't like hardcoding the values or r_SOI for two reasons:
- It is not a physical measurable property, but an artificial radius that in the context of patched conics we decide that works well enough. It is a consequence of other values that we already have, not something you could measure in real life.
- If we calculate it trough a function, we can obtain it for any body with mass and semi-major axis, wether or not its value has been documented before. This means that we can use patched conics for mission planning for anything, from Trappist System to Jovian satellites exploration, or even fictional systems.